Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: build test clean prepare update docker
 
-GO = CGO_ENABLED=0 GO111MODULE=on go
+GO=CGO_ENABLED=1 GO111MODULE=on go
 
 MICROSERVICES=cmd/device-rfid-llrp-go
```

Comment:
Replace all occurrences of `device_rfid_llrp_go` with `device_rfid_llrp`
and `docker_device_rfid_llrp_go` with `device_rfid_llrp`

Reply:
remove below
```
DOCKERS=docker_device_rfid_llrp_go
.PHONY: $(DOCKERS)
```

Reply:
Replaced all and removed what you mentioned